### PR TITLE
Create azorko-bio.md

### PR DESCRIPTION
**Pull request by azorko**
#51

Created azorko-bio.md

### CODE REVIEW

File: azorko-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### azorko
```
Comment (ebuceci) [inline azorko-bio.md:1]: interesting...!

File: azorko-bio.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+- Lives in Cupertino
+- Works in Sunnyvale
```
Comment (kirillrudenko) [inline azorko-bio.md:4]: Cool)

File: azorko-bio.md (hunk 1 above)
Comment (azorko) [inline azorko-bio.md:1]: thanks, I'm quite a writer, I know :D

**Comment by ebuceci** [comment]
:+1: :shipit: and have a :pizza: :tada: !

**Pull request merged**